solutions/simple/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.9.0"
+  required_providers {
+    # Use "greater than or equal to" range in modules
```

Comment:
`# Lock DA into an exact provider version - renovate automation will keep it updated`

Reply:
added